<launch>
  <arg name="robot_ns" default="robot0" />
  <param name="wall_dist" value="0.5" type="double" />
  <remap from="laser_1" to="laser_0" />
</launch>

